packages/azure-kusto-data/source/connectionBuilder.ts

```diff
@@ -216,7 +216,7 @@ export class KustoConnectionStringBuilder {
 
     static withAadDeviceAuthentication(
         connectionString: string,
-        authorityId: string = "common",
+        authorityId: string = "organizations",
```

Comment:
Mind all other APIS take string? authorityId.
Can we make this API Complient or is it a break.

Reply:
Checked the docs - it's equivalent.